_data/packages.yml

```diff
@@ -51,7 +51,13 @@
       github_username: matteobachetti
   all_current_maintainers:
     - name: Daniela Huppenkothen
-      github_username: dhuppenkothen mgullik jdswinbank matteolucchini1
+      github_username: dhuppenkothen
```

Comment:
Ok, maintainers are now populating correctly. We just need a small reminder in the issue to add commas for parsing. If this happens again, we can consider adding more logic to the code it will just get complicated, given the many variations that users could provide.